src/commands/sites/sites-delete.ts

```diff
@@ -1,10 +1,31 @@
 import { OptionValues } from 'commander'
-import inquirer from 'inquirer'
 
-import { chalk, error, exit, log } from '../../utils/command-helpers.js'
+import { chalk } from '../../utils/command-helpers.js'
 import BaseCommand from '../base-command.js'
+import { NetlifyLog, confirm, intro, outro } from '../../utils/styles/index.js'
+
+const deleteSite = async (siteId: string, command: BaseCommand) => {
+  const { api } = command.netlify
+
+  NetlifyLog.message(`Deleting site "${siteId}"...`)
+
+  try {
+    await api.deleteSite({ site_id: siteId })
+  } catch (error_) {
```

Comment:
Is the `_` at the end of the name a typo? 🤔

Reply:
This was a copy and paste from the previous code, but the error function we previously imported that caused us to do this is no longer there so I have updated 👍